### ISSUE TITLE
.ignore list

### ISSUE THREAD

**Issue by namtzigla** [GoogleCloudPlatform/cloud-builders#40]
I'm not sure if this is the place for a feature request for GCCB, so here we go.

If you have a build step like:
```
- name: 'gcr.io/cloud-builders/git'
  args:
    - clone
    - https://github.com/googleapis/googleapis
...
```

and you work on your local computer you obviously have that second repo cloned in the same place but if you submit the build job it will always fail if you don't remove the directory from your local dir.

I think an ignore file list file like .gitignore that would exclude patterns from the uploaded tarball.

Thanks

**Comment by skelterjohn** [comment]
Thanks for the feature request.

This issue tracker is probably the best place to make requests.

Having an ignore file is already on our radar, and we're working with the
gcloud team to make sure we do it in a way that works well with the rest of
the SDK.

**Comment by mpskovvang** [comment]
@skelterjohn any update on this? Has the feature been prioritized?

**Comment by bendory** [comment]
A fix for this will be released in versions > 183.0.0 of the cloud SDK. Version 183 is currently scheduled for release this Wednesday, 12/13/2017.

As you'll see in the documentation when released, `gcloud container builds submit` will obey a `.gcloudignore` file (as will many other `gcloud` commands); if no such file is found, it will obey a `.gitignore` if that file exists.

Please read the documentation in full at `gcloud topic gcloudignore` which is already available in the current production release of the cloud SDK. After Wednesday, there will also be an update posted to the Container Builder documentation at http://cloud.google.com/container-builder/.
